tests/overload.erl

```diff
@@ -565,13 +571,27 @@ get_num_running_gen_fsm(Node) ->
     FsmList = [ proplists:lookup(riak_kv_get_fsm, Call) || Call <- InitCalls ],
     length(proplists:lookup_all(riak_kv_get_fsm, FsmList)).
 
-remote_vnode_queue(Idx) ->
+remote_vnode_gets_in_queue(Idx) ->
```

Comment:
We are assuming that these are only `get` messages we want to count in the queue?